### DISCUSSION TITLE
Are Quarto **books** cite-able?

### DISCUSSION THREAD

**Discussion by agrogan1** [quarto-dev/quarto-cli#3836] [Q&A]
Hello! Is there a way to make a quarto book citeable? The Quarto documentation, [Quarto - Creating Citeable Articles](https://quarto.org/docs/authoring/create-citeable-articles.html), would seem to indicate that this is possible for articles but it does not seem to work for books.

In case it's helpful, my book is here, [Multilevel Thinking](https://agrogan1.github.io/multilevel-thinking/), and my YAML is here, [multilevel-thinking/_quarto.yml at main · agrogan1/multilevel-thinking · GitHub](https://github.com/agrogan1/multilevel-thinking/blob/main/_quarto.yml).

Thank you for any help you can provide.

**Comment by dragonstyle** [comment]
We don't currently support providing the formatted 'appendix' for books - I'm not sure where the logical place to put them would be (particularly given the potential conflict with 'true' appendices that may appear in a book). We do support the `google-scholar` options (as you discovered) which will help with indexing.

What would you think the right thing to do with formatted citation data would be in the case of books (perhaps it belongs on the title page)? It would be helpful to have some ideas of better approaches!

**Comment by agrogan1** [comment]
Thanks for your thoughts!

I was thinking of something that duplicated the citation information that
one can generate in Distill, or I guess in Quarto articles, but you are
right, *where* to put it in a book is a question.

Putting it on the title page is a great idea.

I also think automatically generating an Appendix with citation information
would be useful, but I don't know enough about the internals of Quarto to
know whether this would generate a conflict.

Last thought is that you could let the user put it wherever in the document
they wished, with something like the Quarto `{{< pagebreak >}}` code.
Something like `{{ < citation > }}` ?

Andy

Andrew Grogan-Kaylor
Sandra K. Danziger Collegiate Professor of Social Work
University of Michigan School of Social Work

***@***.***
https://agrogan1.github.io/

**Reply by dragonstyle** [reply]
The user controlled positioning is a really nice idea (we do the same with references using a magic DIV with an ID, which might be a useful pattern here too). I'm going to transfer this to an issue so we don't lose track of the improvement!

**Reply by malcolmbarrett** [reply]
Did an issue get created for this? We also want to be able to generate a citation for our book, and I'd like to keep an eye on this feature

**Reply by mcanouil** [reply]
Yes
<img width="1624" alt="Screenshot 2023-11-01 at 16 18 46" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/6dfd10c9-a3bb-4b8a-8d2d-35ea769ec304">

- https://github.com/quarto-dev/quarto-cli/issues/3838

**Reply by malcolmbarrett** [reply]
Thanks! I've never noticed that part of the page

**Comment by agrogan1** [comment]
Great. Thanks!

On Tue, Jan 3, 2023 at 2:34 PM Charles Teague ***@***.***>
wrote:

> The user controlled positioning is a really nice idea (we do the same with
> references using a magic DIV with an ID, which might be a useful pattern
> here too). I'm going to transfer this to an issue so we don't lose track of
> the improvement!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/3836#discussioncomment-4585099>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGG5FDOSLOKF6KWWCUQT4A3WQR5MTANCNFSM6AAAAAATP2YSIU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 

Sent from Gmail Mobile
Andrew Grogan-Kaylor
***@***.***
https://agrogan1.github.io/
